go-concourse/concourse/users.go

```diff
@@ -0,0 +1,10 @@
+package concourse
+
+import (
+	"github.com/concourse/concourse/atc"
+	"time"
+)
+
+func (client *client) ListActiveUsersSince(since time.Time) ([]atc.User, error) {
```

Comment:
Is this used somewhere?

Reply:
This is for `fly` and it's WIP.  We are planning to have a fly command that can print out all active users or some of the users since the given date.